azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md

```diff
@@ -99,7 +101,7 @@ The following firewall URL exceptions are needed on all servers in the Azure Sta
 
 ## Install PowerShell modules and update extensions
 
-To prepare to install Azure Arc Resource Bridge on an Azure Stack HCI cluster and create a VM cluster-extension, perform these steps:
+To prepare to install Azure Arc Resource Bridge on an Azure Stack HCI cluster and create a VM cluster-extension, perform these steps (throught RDP or console session, remote Powershell is not supported):
```

Comment:
Typo in this sentence, "throught" is not a word.

Reply:
Sorry, through vs throught.